### ISSUE TITLE
Figure out why this warning is shown

### ISSUE THREAD

**Issue by NaasvanH** [TheProfitTable/loanportr#23]
Warning in as.POSIXlt.POSIXct(x, tz) :
  unknown timezone 'zone/tz/2017c.1.0/zoneinfo/Africa/Johannesburg'

**Comment by NaasvanH** [comment]
And when installing package:   

```Warning in strptime(xx, f <- "%Y-%m-%d %H:%M:%OS", tz = tz) :
  unknown timezone 'zone/tz/2017c.1.0/zoneinfo/Africa/Johannesburg'
ERROR: failed to lock directory '/Library/Frameworks/R.framework/Versions/3.4/Resources/library' for modifying
Try removing '/Library/Frameworks/R.framework/Versions/3.4/Resources/library/00LOCK-loanportr'
Warning in install.packages :
  installation of package '/Users/naasvanheerden/TPT_R_Projects/loanportr_0.1.7.tar.gz' had non-zero exit status```

**Comment by DandeVilliers** [comment]
Yes this is the timezone warning I was talking about the other day 🤔

**Comment by floresfdev** [comment]
Hi. Are you using macOS High Sierra? I've just stumbled upon some reports of the issue and a workaround: https://stackoverflow.com/questions/47314121/r-error-unknown-timezone-with-as-posixct/47467017#47467017

`Sys.setenv(TZ = "Africa/Johannesburg")` should solve it for the moment.

**Comment by NaasvanH** [comment]
Yes, we are both using macOS High Sierra.   
Tried it on my side and it works.  
Thanks @floresfdev !

**Comment by floresfdev** [comment]
R 3.4.3 should take care of it: https://stat.ethz.ch/pipermail/r-announce/2017/000623.html

> A workaround has been added for the changes in location of -zone files in macOS 10.13 'High Sierra' and again in 10.13.1, so the default time zone is deduced correctly from the system setting when R is configured with --with-internal-tzcode (the default on macOS).

**Comment by DandeVilliers** [comment]
Yeah, seems to be fixed with R 3.4.3. Thanks @floresfdev
